backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4825,3 +4825,14 @@ databaseChangeLog:
           - sql:
               sql: |
                 DROP TYPE ${database.defaultSchemaName}.TEST_RESULT_UPLOAD_ERROR;
+  - changeSet:
+      id: result_upload_error-add-no-phi-user
+      author: nathan@skylight.digital
+      comment: Adds permission for the no-PHI user to select from the bulk test result upload errors
+      changes:
+        - tagDatabase:
+            tag: results_upload_error-add-no-phi-user
+        - sql:
+            sql: GRANT SELECT ON ${database.defaultSchemaName}.result_upload_error TO ${noPhiUsername};
+      rollback:
+            sql: REVOKE SELECT ON ${database.defaultSchemaName}.result_upload_error FROM ${noPhiUsername};
```

Comment:
The rollback block is invalid

Reply:
Are you looking at an out-of-date commit, maybe? I seem to be able to roll this change back:

```
➜  backend git:(nac/6153-add-result-upload-errors-metabase) ✗ ./gradlew liquibaseRollbackCount -PliquibaseCommandValue=1

> Task :liquibaseRollbackCount
liquibase-plugin: Running the 'main' activity...

Starting Liquibase at 11:39:59 (version 4.9.1 #1978 built at 2022-03-28 19:39+0000)
[2023-07-24 11:40:00] INFO [liquibase.lockservice] Successfully acquired change log lock
[2023-07-24 11:40:00] INFO [liquibase.changelog] Reading from simple_report.databasechangelog
Rolling Back Changeset: db/changelog/db.changelog-master.yaml::result_upload_error-add-no-phi-user::nathan@skylight.digital
[2023-07-24 11:40:01] INFO [liquibase.lockservice] Successfully released change log lock
Liquibase: Rollback has been successful.
```

Reply:
hmm... can you verify the access has been revoked?

Reply:
This is the error I am seeing
![image](https://github.com/CDCgov/prime-simplereport/assets/4952042/89918acc-fdd8-4b89-9a1a-d571aa26302d)
and This is what is should be 
![image](https://github.com/CDCgov/prime-simplereport/assets/4952042/0c0716c8-5c47-4fe6-a61f-f7f8823eb355)

Reply:
I can deploy it out to a Metabase environment to be certain, but looking at the DDL definitions for the table it does seem like the permissions are being changed as expected:

_Migration up_
<img width="607" alt="Screen Shot 2023-07-24 at 11 54 18 AM" src="https://github.com/CDCgov/prime-simplereport/assets/27730981/62bf5759-bbb5-4e1b-8956-b9fee5d44604">

_Migration down_
<img width="612" alt="Screen Shot 2023-07-24 at 11 53 26 AM" src="https://github.com/CDCgov/prime-simplereport/assets/27730981/4f977ba9-45ea-4e76-8d68-b7209a02a86c">

@zdeveloper what are you seeing?

Reply:
hmm I am a bit puzzled, I think its removing the granting migration, thats why you dont see `grant select on result_upload_error to simplereport_no_phi` in the migration down DDL

I would expect it to have the revoke statement in the DDL in the migration down


`REVOKE SELECT ON result_upload_error FROM simple_report_no_phi`

Reply:
And also my understanding how the DDL for permissions was wrong, I expected it to have the grant and the revoke statement in the DDL, but I guess liquibase was able to eliminate them both from the DDL.